homeassistant/components/advantage_air/entity.py

```diff
@@ -9,7 +14,9 @@
 class AdvantageAirEntity(CoordinatorEntity):
     """Parent class for Advantage Air Entities."""
 
-    def __init__(self, instance, ac_key, zone_key=None):
+    def __init__(
+        self, instance: advantage_air, ac_key: int, zone_key: str | None = None
```

Comment:
ac_key is a string
https://github.com/home-assistant/core/blob/dev/tests/components/advantage_air/fixtures/getSystemData.json#L3